### ISSUE TITLE
gcr.io/google_containers/hyperkube-arm:v1.2.3 missing

### ISSUE THREAD

**Issue by rhuss** [kubernetes/kubernetes#24785]
Any chance for a v1.2.3 of the intrastructure containers in gcr.io ? (hope this is the right place to ask)

```
curl https://gcr.io/v2/google_containers/hyperkube-arm/tags/list | jq .
```

yields

```yaml
{
  "manifest": {
    "sha256:087b6d29c288a87c9b3729bcfbec94a5d4663aba127bd91f858da4f7859c81f5": {},
    "sha256:2625432de438ed34f35df2b65a97153a94dd43762cab87dbf080cc12362562c4": {},
    "sha256:969ef34ed87a846a4551a889130a4466be299df2f20dadeb96b2ef3b9d18d947": {
      "tag": [
        "v1.3.0-alpha.2"
      ]
    },
    "sha256:d1b0b41b4e217649f8f5e0ab6e968329b6b34abd3ad5ad997b1e4157ac207b07": {
      "tag": [
        "v1.1.2"
      ]
    },
    "sha256:f5f044009c668468653a3f439acc22abf7d1e8c27ed8f4ce07c8350654646ac4": {
      "tag": [
        "v1.2.0"
      ]
    },
    "sha256:fb5912373dd4e57d518e2534680007352cd30b21263188e12ae0e3062c3654a9": {
      "tag": [
        "v1.2.0-beta.1"
      ]
    }
  },
  "name": "google_containers/hyperkube-arm",
  "tags": [
    "v1.1.2",
    "v1.2.0",
    "v1.2.0-beta.1",
    "v1.3.0-alpha.2"
  ]
}
```

**Comment by luxas** [comment]
Hi @rhuss!
You should have pinged me from the start :), but noticed this now.
I've written the code for `hyperkube-arm`, and you'll find all links in #17981 

For `v1.2.0`, binaries are cross-compiled for ARM on `make release` thanks to #19769, but Docker images aren't built (there wasn't time for that before the release)

Then I submitted #21617, so `v1.3.0-alpha.2` and over will be automatically pushed on every release.
I also made this for `arm64` and `ppc64le`: #22149

TL;DR; 
The code above will not be cherry-picked into `v1.2`, but `v1.3` will feature `arm`, `arm64` and `ppc64le` images that is usable with the `docker-multinode` (`etcd-arm` and `flannel-arm` are also pushed) deployment, and the DNS addon is also cross-platform.

But if you're only looking for binaries, fetch them here:
```
# Any other version v1.2.0-alpha.8+ works as well
export K8S_VERSION=v1.2.4
curl -sSL https://storage.googleapis.com/kubernetes-release/release/${K8S_VERSION}/bin/linux/arm/kubectl > /usr/bin/kubectl
curl -sSL https://storage.googleapis.com/kubernetes-release/release/${K8S_VERSION}/bin/linux/arm/hyperkube> /usr/bin/hyperkube
```

**Comment by rhuss** [comment]
@luxas thanks a lot for your awesome work ;-) I'm perfectly happy with going directly to v1.3.

Thanks !
